src/ui/pages/WalletSelect/WalletSelect.tsx

```diff
@@ -170,6 +196,44 @@ export function WalletSelect() {
             }}
             selectedAddress={singleAddress}
             showAddressValues={true}
+            renderItemFooter={({ wallet }) => {
+              const walletMeta = walletsMeta?.find(
+                (meta) =>
+                  normalizeAddress(meta.address) ===
+                  normalizeAddress(wallet.address)
+              );
+              const addWalletParams = getWalletParams(wallet);
+
+              const exploreRewardsUrl = walletMeta?.membership.newRewards
+                ? `${ZERION_ORIGIN}/rewards?section=rewards&${addWalletParams}`
```

Comment:
https://github.com/zeriontech/zerion-wallet-extension/pull/657/files#r1836574324

Reply:
updated ✅ could you please verify?